src/data/fullness.txt

```diff
@@ -8,8 +8,8 @@
 [glitch season reward name]	0	0		0	0	0	0	Inedible, starts a fight with %monster%
 &quot;caramel&quot; orange	1	1	awesome	4	0	0	0	gain 10 turns of a random positive effect
 &quot;meat&quot; stick	1	5	good	3	15-25	0	0	50 Empty Inside (+50 Food Drop, +50 Booze Drop)
-20-lb can of rice and beans	15	1	good	45	0	0	0	100 Stuffed (+100 HP)
 3vi1 pr0n m4nic0tti	4	3	awesome	11-16	0	0	60-63	10 A Little Bit Evil (+2 ML)
+20-lb can of rice and beans	15	1	good	45	0	0	0	100 Stuffed (+100 HP)
```

Comment:
2 should probably come before 3?

Reply:
It's natural sorted - 20 is coming after 3